ts/receiver/receiver.ts

```diff
@@ -705,7 +608,7 @@ export async function handleMessageEvent(event: any): Promise<void> {
   const shouldSendReceipt =
     isIncoming &&
     data.unidentifiedDeliveryReceived &&
-    !data.friendRequest &&
+    !data.isSessionRequest &&
```

Comment:
```
// Note(LOKI): don't send receipt for FR as we don't have a session yet
```
that comment needs to be removed.

also we don't need to worry if a message `isSessionRequest` or not down here. If a message made it down here then we wanted it to be shown in the UI.

Reply:
well before we were sending receipt messages on endSession anyway, since those messages were not friend requests